login/tests.py

```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
+
```

Comment:
PR without any tests? 
Please add some login check

Reply:
Please consider to add here tests
If you have login page at lease add test to verify it is working

Reply:
This is a pr only for initialization of the login app, without any login form.
I will add the different pr with the login form, and will add tests to it.

Reply:
@amittcohen You can add this simple test: https://github.com/redhat-beyond/JobSeeker/pull/55#pullrequestreview-920744972

Reply:
@amittcohen 
Please add a test, it is mandatory for Merge
As per taljacobs's sample